src/main/java/co/cask/gcp/speech/SpeechToTextTransform.java

```diff
@@ -157,46 +96,39 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
     Schema recordSchema = getSchema(schema);
     List<StructuredRecord> speechArray = new ArrayList<>();
```

Comment:
I don't think the output should contain an array of possibilities. It makes it very difficult to chain to anything else. Instead, let's just use the first (most likely) result. We can have optional 'transcriptField' and 'confidenceField' properties that default to 'transcript' and 'confidence'.

We should also have a Get Schema button that takes the input schema and adds these fields